tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "target": "es6",
+        "target": "es2017",
```

Comment:
What benefits are gained by targeting es2017 instead of es6?

Reply:
String.padStart and String.padEnd

Reply:
In the future, please do not combine changes like this with a feature addition but make separate PRs